services/registration/src/routes/application.ts

```diff
@@ -1,10 +1,43 @@
 import { asyncHandler } from "@api/common";
 import express from "express";
+import Ajv from "ajv";
+import addFormats from "ajv-formats";
 
 import { ApplicationModel } from "../models/application";
 
 export const applicationRouter = express.Router();
 
+const ajv = new Ajv();
+addFormats(ajv);
+
+const branchSchema = {
+  type: "object",
+  properties: {
+    name: { type: "string" },
+    type: { type: "string" },
+    settings: {
+      type: "object",
+      properties: {
+        open: { type: "string", format: "date-time" },
+        close: { type: "string", format: "date-time" },
+      },
+      required: ["open", "close"],
+    },
+    jsonSchema: { type: "object", default: {} },
+    uiSchema: { type: "object", default: {} },
+  },
+  required: ["name", "type", "settings", "jsonSchema", "uiSchema"],
+};
```

Comment:
So for validating the application data, we don't need the entire branch schema here. We should use the `jsonSchema` field from the branch model to get the validation schema, and then just validate the application data against that. Does that make more sense?